gordrick/storemanager_api2
# [#161422246] Write failing tests for the API endpoints

**What does this PR do**?
This PR creates tests for all the project specified endpoints.

**Description of Task to be completed:**
Write tests for the API endpoints

**How to manually test:**

Clone the repository:`$ git clone https://github.com/gordrick/storemanager_api2.git`
cd into the repository after cloning
`$ git checkout ch-write-tests-161422246 to switch into the testing branch`
`$ pip install nose`
`$ nosetests`

The tests should fail since no code has been written yet.
**Relevant pivotal tracker story**
[161422246](https://www.pivotaltracker.com/projects/2203025)
